### PR TITLE
use models.UUIDField

### PR DESCRIPTION
**Pull request by zebuline**
* drop support of django < 1.8
* use models.UUIDField

### CODE REVIEW

**Comment by zebuline** [comment]
ping @tbcvl @k4nar @brunobord

File: CHANGELOG [hunk 1]
```diff
@@ -5,10 +5,14 @@ This document describes changes between each past release. For information
 about future releases, check `milestones`_ and :doc:`/about/vision`.
 
 
-0.5 (unreleased)
+1.0 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- Feature #21 - Drop support of Django 1.5, 1.6 and 1.7
+  Explicitely mark Django 1.5, 1.6 and 1.7 as not supported
```
Comment (k4nar) [inline CHANGELOG]: Explicitly* :)

**Comment by k4nar** [comment]
LGTM :+1:

**Comment by tbcvl** [comment]
LGTMT

**Pull request merged**